docs/self-managed/console-deployment/overview.md

```diff
@@ -0,0 +1,13 @@
+---
+id: overview
+title: "Console (Self-Managed)"
+sidebar_label: "Overview"
+description: ""
+---
+
+:::note
+Camunda Console Self-Managed is an [Alpha release](/reference/alpha-features.md). It is in an early state for you to test and participate in development by sharing your feedback.
```

Comment:
Is there a particular source you want to direct users for feedback? If not, we can direct them to the `/contact` page.

Reply:
Done. Added a feedback page we are settng up for Alpha